suites/nautilus/ansible/sanity_containerized_ceph_ansible.yaml

```diff
@@ -264,6 +264,7 @@ tests:
 
    - test:
       name: Check for old container directories
+      polarion-id: CEPH-83573507
```

Comment:
added polarion id for https://github.com/red-hat-storage/cephci/pull/144

Reply:
@Ameenasuhani can you send a separate pr for this ? It's not related to upgrade scenario

Reply:
@XDsubhash Okay, I've made the changes